packages/manager-react-components/src/components/templates/onboarding/onboarding.component.tsx

```diff
@@ -38,26 +44,49 @@ const OnboardingLayoutButton: React.FC<OnboardingLayoutButtonProps> = ({
   onOrderButtonClick,
   onmoreInfoButtonClick,
   isActionDisabled,
+  orderIam,
 }) => {
   if (!orderButtonLabel && !moreInfoButtonLabel) {
     return <></>;
   }
   return (
     <div className="flex flex-col gap-3 sm:gap-4 w-full sm:w-fit sm:flex-row sm:items-center sm:justify-center">
-      {orderButtonLabel && (onOrderButtonClick || orderHref) && (
-        <OdsButton
-          className="[&::part(button)]:w-full sm:w-auto"
-          size={ODS_BUTTON_SIZE.md}
-          onClick={() => {
-            onOrderButtonClick?.();
-            if (orderHref) {
-              window.open(orderHref, '_blank');
-            }
-          }}
-          label={orderButtonLabel}
-          isDisabled={isActionDisabled}
-        />
-      )}
+      {(orderIam?.iamActions || orderIam?.iamActions?.length > 0) &&
+        orderButtonLabel &&
+        (onOrderButtonClick || orderHref) && (
+          <ManagerButton
+            id="orderButton"
+            className="[&::part(button)]:w-full sm:w-auto"
+            size={ODS_BUTTON_SIZE.md}
+            onClick={() => {
+              onOrderButtonClick?.();
+              if (orderHref) {
+                window.open(orderHref, '_blank');
+              }
+            }}
+            label={orderButtonLabel}
+            isDisabled={isActionDisabled}
+            urn={orderIam.urn}
```

Comment:
what about 
```suggestion
            {...(orderIam ? orderIam : {}) }
```

Reply:
@sachinrameshn with this you shouldn't need to have 2 declaration of the `ManagerButton` no ?

Reply:
One is **ManagerButton** and another one is **odsButton** @tristanwagner I need **ManagerButton** when we have IAM.